docs/tutorial/flask.rst

```diff
@@ -0,0 +1,530 @@
+=================================================
+Write your first Kubernetes charm for a Flask app
+=================================================
+
+Imagine you have a Flask application backed up by a database
+such as PostgreSQL and need to deploy it. In a traditional setup,
+this can be quite a challenge, but with Juju you’ll find yourself
+deploying, configuring, scaling, integrating, monitoring, etc.,
+your Flask application in no time. Let’s get started!
+
+In this tutorial we will build a rock and Kubernetes charm for a
+Flask application using the charm SDK, so we can have a Flask
+application up and running with Juju in about 90 minutes.
+
```

Comment:
Any chance we could include an estimated completion time here? I'm fond of those in tutorials.

Reply:
The 90 minute time was meant to be the estimated completion time. Do you mean something else, or should it be rephrased?

Reply:
Sorry, I got myself mixed up. I meant to ask if can we include the estimate on a separate line, to visually mimic how some tutorials outside of Canonical include time as a pill under the document title. Basically, I think it's worthwhile to draw more attention to it on its own line.

Reply:
Made the change